conf/messages.cy

```diff
@@ -1965,6 +1965,16 @@ optout.confirmOptOut.info-message       = Yn y dyfodol, efallai y bydd gofyn i c
 optout.confirmOptOut.confirm            = Cadarnhau
 optout.confirmOptOut.cancel             = Canslo
 
+#opt out multi year check answers
+optout.checkAnswers.heading             = Gwiriwch eich atebion
+optout.checkAnswers.optOut              = Optio allan o
+optout.checkAnswers.taxYears            = flwyddyn dreth {0} i {1} ymlaen
+optOut.checkAnswers.change              = Newid
```

Comment:
'Change' and 'Cancel' are not shown in Welsh see screensh
![Screenshot 2024-06-12 at 12 22 22](https://github.com/hmrc/income-tax-view-change-frontend/assets/165788042/d2786660-962e-4da3-bf8f-a572cbd89c9b)
ot